code/datums/diseases/advance/advance.dm

```diff
@@ -280,8 +280,10 @@
 			severity = DISEASE_SEVERITY_DANGEROUS
 		if(5)
 			severity = DISEASE_SEVERITY_BIOHAZARD
-		if(6 to INFINITY)
+		if(6 to INFINITY && !99)
```

Comment:
I am like 90% sure this doesn't work. Have you tested this?

Reply:
It does not